playbooks/common/openshift-cluster/config.yml

```diff
@@ -1,22 +1,32 @@
 ---
 # TODO: refactor this into its own include
 # and pass a variable for ctx
-- name: Verify Requirements
-  hosts: oo_all_hosts
-  roles:
-  - openshift_health_checker
-  vars:
-  - r_openshift_health_checker_playbook_context: install
-  post_tasks:
-  - action: openshift_health_check
-    args:
-      checks:
-      - disk_availability
-      - memory_availability
-      - package_availability
-      - package_version
-      - docker_image_availability
-      - docker_storage
+- name: Verify Install Requirements
+ hosts: oo_all_hosts
```

Comment:
There is an indentation error here, needing to prepend an extra space here and in the rest of the block.